src/studio-home/tabs-section/libraries-v2-tab/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Icon, Row, Pagination } from '@openedx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import useListStudioHomeV2Libraries from '../../data/apiHooks';
+import { LoadingSpinner } from '../../../generic/Loading';
+import AlertMessage from '../../../generic/alert-message';
+import CardItem from '../../card-item';
+import messages from '../messages';
+
+const LibrariesV2Tab = ({
+  libraryAuthoringMfeUrl,
+  redirectToLibraryAuthoringMfe,
+}) => {
+  const intl = useIntl();
+
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handlePageSelect = (page) => {
+    setCurrentPage(page);
+  };
+
+  const {
+    data,
+    isLoading,
+    isError,
+  } = useListStudioHomeV2Libraries({ page: currentPage });
+
+  if (isLoading) {
+    return (
+      <Row className="m-0 mt-4 justify-content-center">
+        <LoadingSpinner />
+      </Row>
+    );
+  }
+
+  const libURL = (id) => (
+    libraryAuthoringMfeUrl && redirectToLibraryAuthoringMfe
+      ? `${libraryAuthoringMfeUrl}library/${id}`
+      // Redirection to the placeholder is done in the MFE rather than
+      // through the backend i.e. redirection from cms, because this this will probably change
+      : `${window.location.origin}/course-authoring/library/${id}`
+  );
+
+  return (
+    isError ? (
+      <AlertMessage
+        title={intl.formatMessage(messages.librariesTabErrorMessage)}
+        variant="danger"
+        description={(
+          <Row className="m-0 align-items-center">
+            <Icon src={Error} className="text-danger-500 mr-1" />
+            <span>{intl.formatMessage(messages.librariesTabErrorMessage)}</span>
+          </Row>
+        )}
+      />
+    ) : (
+      <div className="courses-tab-container">
+        <div className="d-flex flex-row justify-content-between my-4">
+          {/* Temporary div to add spacing. This will be replaced with lib search/filters */}
+          <div className="d-flex" />
+          <p data-testid="pagination-info">
+            {intl.formatMessage(messages.coursesPaginationInfo, {
+              length: data.results.length,
+              total: data.count,
+            })}
+          </p>
+        </div>
+
+        {
+          data.results.map(({
+            id, org, slug, title,
+          }) => (
+            <CardItem
+              key={`${org}+${slug}`}
+              isLibraries
+              displayName={title}
```

Comment:
The [ContentLibrary model](https://github.com/openedx/edx-platform/blob/51ebabfd96b1d3c0d096c46981cc09e17ef11388/openedx/core/djangoapps/content_libraries/models.py#L84) doesn't have a `title` field.. I'm not sure what should go here instead?

Reply:
That's right, however the [serializers](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_libraries/serializers.py#L23-L61) have the `title` field.

Here is an example of what is returned in the from the API:

```js
    {
        "id": "lib:SampleTaxonomyOrg1:TL1",
        "type": "complex",
        "org": "SampleTaxonomyOrg1",
        "slug": "TL1",
        "title": "Test Library 1",
        "description": "",
        "num_blocks": 0,
        "version": 0,
        "last_published": null,
        "allow_lti": false,
        "allow_public_learning": false,
        "allow_public_read": false,
        "has_unpublished_changes": false,
        "has_unpublished_deletes": false,
        "license": ""
    }
```

Reply:
Oh my mistake, you're totally correct, there is a `title` field in the serializer, and it's [the library's learning package title](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_libraries/api.py#L284). But since I was naughty and neglected to use the `content_libraries.api` to create my libraries, they didn't have learning packages.

My bad!

Reply:
ohh I see, gotcha!